services/vbox-service.jsx

```diff
@@ -0,0 +1,8 @@
+
+
+const isNonWindow = typeof window === 'undefined';
```

Comment:
Why is this `.jsx`? Can you move this into `src/`?

Reply:
Done

Reply:
@hbeckeri Actually if I don't use `jsx` then it does not export `VBox` properly. Changed back to `jsx`